tests/test_modeling_common.py

```diff
@@ -3887,8 +3885,8 @@ def test_sdpa_can_compile_dynamic(self):
             self.skipTest(reason="This test requires an NVIDIA GPU with compute capability >= 8.0")
         elif device_type == "rocm" and major < 9:
             self.skipTest(reason="This test requires an AMD GPU with compute capability >= 9.0")
-        else:
-            self.skipTest(reason="This test requires a Nvidia or AMD GPU")
+        elif device_type not in ["cuda", "rocm", "xpu"]:
```

Comment:
there's no device type "rocm"

Reply:
`get_device_properties` utility will return device type `rocm` as the code [here](https://github.com/huggingface/transformers/blob/main/src/transformers/testing_utils.py#L3107), and the `device_type` variable is the return value of `get_device_properties`.